### ISSUE TITLE
Custom message for the maze

### ISSUE THREAD

**Issue by aroberge** [aroberge/reeborg#478]
When escaping the maze, instead of the standard message
![image](https://github.com/user-attachments/assets/29f26675-e673-4ee2-9854-244c397a9ed4)
it might be useful to have a custom one, like "Congratulations, you have successfully escaped the maze."

**Comment by sjlearmonth** [comment]
I don't even get "Reeborge says. I'm done!".

**Comment by aroberge** [comment]
> I don't even get "Reeborge says: I'm done!".

That's because the solution requires Reeborg to stop when stepping out of the maze.

**Comment by sjlearmonth** [comment]
Yes he does stop right on top of the flag but I still don’t get a message.Sent from my iPhoneStephen J LearmonthMobile: 07757 782537On 19 Aug 2024, at 18:51, André Roberge ***@***.***> wrote:﻿

I don't even get "Reeborge says: I'm done!".

That's because the solution requires Reeborg to stop when stepping out of the maze.

**Comment by sjlearmonth** [comment]
﻿Do I need to use the test condition “at_goal()” to decide when to terminate the program?Sent from my iPhoneStephen J LearmonthMobile: 07757 782537On 19 Aug 2024, at 18:59, ***@***.*** wrote:﻿Yes he does stop right on top of the flag but I still don’t get a message.Sent from my iPhoneStephen J LearmonthMobile: 07757 782537On 19 Aug 2024, at 18:51, André Roberge ***@***.***> wrote:﻿

I don't even get "Reeborge says: I'm done!".

That's because the solution requires Reeborg to stop when stepping out of the maze.

**Comment by aroberge** [comment]
Yes, you should test that condition. If it is satisfied, you can always use `done()` to end the program.
